### PR TITLE
Add portfolio projects, "works section" and about me section

### PR DESCRIPTION
**Pull request by Erick-Osterling**
To add the new sections, a combination of the tools of "Grid" and "Flexbox" was used. The style was acquired from the "Figma" webpage (template 3).

Buttons have hover and enabled features.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
     </ul>
     
   </section>   
-        
-   
+  
+  <section class="intro-works">
+
+    <button class="deco-button"><img class="img-arrow" src="images/arrow-dropdown.jpg" alt="arrow dropdown"></button>
+    <h1 class="works-text">My Recent Works</h1>
```
Comment (Alexoid1) [inline index.html]: - [x] Replace the h1 with a h2. The h1 tag is just for the title of the website.

File: index.html [hunk 2]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
...
+
+
+<!--Grid container with all 6 work templates--> 
+
+  <div class="grid-container">
+
+    <section class="works-template work1">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace `div ` with a `ul` and use the `li` tag to list the technologies used in the project.

File: index.html [hunk 3]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
...
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work2">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace `div ` with a `ul` and use the `li` tag to list the technologies used in the project.

File: index.html [hunk 4]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
...
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work3">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace `div ` with a `ul` and use the `li` tag to list the technologies used in the project.

File: index.html [hunk 5]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
...
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work4">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace `div ` with a `ul` and use the `li` tag to list the technologies used in the project.

File: index.html [hunk 6]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
...
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
     
+    <section class="works-template work5">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace `div ` with a `ul` and use the `li` tag to list the technologies used in the project.

File: index.html [hunk 7]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
...
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
     
+    <section class="works-template work6">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
```
Comment (Alexoid1) [inline index.html]: - [x] Kindly replace `div ` with a `ul` and use the `li` tag to list the technologies used in the project.

File: index.html [hunk 8]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
...
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+
+  </div>
+
+<!--About me section-->
+  <section class="aboutme-container">
+    <h1 class="aboutme-h1">About me</h1>
```
Comment (Alexoid1) [inline index.html]: - [x] Replace the `h1` with an `h2`. The h1 tag is just for the title of the website.

File: index.html [hunk 9]
```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
...
+    <p class="aboutme-p"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p> 
+    <button class="aboutme-button">Get My Resume</button>
```
Comment (Alexoid1) [inline index.html:144]: - [x] Kindly add more width to the button to make it look just like the Figma design.
Captura de pantalla_2021-09-18_01-20-48

**Figma**
Captura de pantalla_2021-09-18_01-21-04

File: styles2.css [hunk 10]
```diff
@@ -39,49 +38,38 @@ nav:hover {
 division between top bar and the rest
 */
 
-section {
+.port-front {
   background-image: url("images/Ilustration-mobile.png");
```
Comment (Alexoid1) [inline styles2.css]: - [ ] The background image it seems that not to be there in the mobile version. Make the image responsive in order to make your design the same as the Figma template.
Captura de pantalla_2021-09-18_01-30-03

**Figma**
Captura de pantalla_2021-09-18_01-30-57

---------------------------------------------------------------------------
Captura de pantalla_2021-09-18_01-30-29

**Figma**
Captura de pantalla_2021-09-18_01-31-25

File: index.html [hunk 11]
```diff
@@ -30,7 +31,7 @@ <h2 class="secondinfo">I’m a software developer</h2>
     <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
```
Comment (Alexoid1) [inline index.html]: - [ OPTIONAL] Use the `font-size` values same as the Figma template.

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @Erick-Osterling ,

Impressive project!
You are doing a great job so far, and the project is looking good.

## Required Changes :recycle:
 - Kindly add a link with a live version of the project in the README.md file.

For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by carlos-ssh — APPROVED** [review]
Hi @Erick-Osterling , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**